src/main/java/org/mule/extension/file/internal/command/LocalCopyCommand.java

```diff
@@ -60,6 +61,6 @@ protected void doExecute(Path source, Path targetPath, boolean overwrite, CopyOp
    */
   @Override
   protected String getAction() {
-    return "copying";
+    return "copy";
```

Comment:
this affects the message thrown at org.mule.extension.file.internal.command.AbstractLocalCopyCommand#execute:90. That been said, that message is not good and should also be fixed